README.md

```diff
@@ -33,9 +17,9 @@ For Dev Support:
 
 The following is a list of awesome apps built on top of the Subsocial protocol. 
 
-### Contents
+## Contents
```

Comment:
I think i saw some plugins or built-in functionality for the generation of ToC for README files.

Reply:
https://github.blog/changelog/2021-04-13-table-of-contents-support-in-markdown-files/

Reply:
Do you think it's easily discoverable to people?
Like I new this feature but I rarely use it.